### PR TITLE
Adds events API to admin/events

### LINKED ISSUE #631: Implement API to admin/events
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

- Implement events API to admin/events

### PR DESCRIPTION
**Pull request by geekyd**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Adds events API to admin/events
- Reflect the page size in the page-size dropdown

Fixes #631

### CODE REVIEW

**Comment by niranjan94** [comment]
Deployment link: https://pr-637-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-637-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-637-open-event-frontend-fossasia.surge.sh

File: app/routes/admin/events/list.js [hunk 1]
```diff
@@ -17,205 +18,75 @@ export default Route.extend({
...
-        total : 10
-      }]
-    }];
+      ];
+    } else {
+      filterOptions = [];
+    }
+
+    return this.store.query('event', {
+      include      : 'tickets,sessions,speakers',
```
Comment (iamareebjamal) [inline app/routes/admin/events/list.js]: This is where it all started

**Pull request merged**